src/test/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/entity/CardTest.java

```diff
@@ -52,4 +59,34 @@ void compare_smaller_greater_test() {
     assertFalse(card_1.isGreaterThan(card_2));
     assertTrue(card_2.isGreaterThan(card_1));
   }
+
+  @Test
+  void is_not_highest_card_when_round_is_null() {
+    Card card = new Card();
+
+    assertThat(card.isHighestCardInRound(), is(false));
+  }
+
+  @Test
+  void is_not_highest_card_when_this_not_in_highest_cards() {
+    Round round = mock(Round.class);
+    when(round.getHighestCards()).thenReturn(new LinkedHashSet<>(Set.of(new Card())));
+    Card card = new Card();
+    card.setRound(round);
+
+    assertThat(card.isHighestCardInRound(), is(false));
+  }
+
+  @Test
+  void is_highest_card_when_this_is_in_highest_cards() {
+    Round round = mock(Round.class);
+    Card card = new Card();
+    when(round.getHighestCards()).thenReturn(new LinkedHashSet<>(Set.of(card)));
+    card.setRound(round);
+
+    assertThat(card.isHighestCardInRound(), is(true));
+
+    when(round.getHighestCards()).thenReturn(new LinkedHashSet<>(Set.of(card, new Card())));
+    assertThat(card.isHighestCardInRound(), is(true));
+  }
```

Comment:
Die Methode getHighestCards() wird mit diesen Tests aber gar nicht getestet?

Reply:
ne, es wird eigentlich nur der null check und LinkedHashSet::contains getestet.
Du testest ja getHighestCards in HandTest

Reply:
HandTest?

Reply:
Sorry, natürlich RoundTest